auth0_flutter_platform_interface/lib/src/method_channel_auth0_flutter_auth.dart

```diff
@@ -52,6 +57,39 @@ class MethodChannelAuth0FlutterAuth extends Auth0FlutterAuthPlatform {
     return Challenge.fromMap(result);
   }
 
+
+  @override
+  Future<void> passwordlessWithEmail(
```

Comment:
This one should match the name of the public method for simplicity and ease of future maintenance (less confusing for someone making changes in the future).